docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<p>I think the tutorial is awesome! I just provided a few comments. Btw, it might be a good idea to click on "Resolve Conversation" after we discuss a certain comment. This way you and I can see what comments are still needed to be discussed further. Does that sound good? </p><p> </p><p>(btw, I already resolved the ones that were previously discussed).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
@ken-maeda 
I think you can see the "Resolve Conversation" in `ReivewNB`. So, just click on the purple button `ReivewNB`, located at the top of this webpage.